src/main/java/com/exadel/sandbox/officeFloorAttachment/dto/attachmentDto/AttachmentBaseDto.java

```diff
@@ -0,0 +1,15 @@
+package com.exadel.sandbox.officeFloorAttachment.dto.attachmentDto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AttachmentBaseDto {
+    private String chatId;
+    private Long messageId;
+    private String atName;
```

Comment:
please do not use prefix in DTOs, check other places

Reply:
I don't understand what you mean on prefix can you explain more clearly please